src/rpc/CMakeLists.txt

```diff
@@ -11,13 +11,13 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-# Authored by: Alberto Aguirre <alberto.aguirre@canonical.com>
 
 set(MULTIPASS_PROTOCOL_SPEC multipass.proto)
 set(GRPC_GENERATED_SOURCE_DIR ${MULTIPASS_GENERATED_SOURCE_DIR}/multipass/rpc)
 file(MAKE_DIRECTORY ${GRPC_GENERATED_SOURCE_DIR})
 
+add_compile_options(-Wno-error)
```

Comment:
```suggestion
add_compile_options(-Wno-error=pedantic)
```

Reply:
Ah, cool, didn't realize that it could be that specific.